### PR TITLE
update kernel-mshv

### PR DESCRIPTION
**Pull request by Redent0r**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Bump Kata components to match lsg release v2302.8.1

###### Change Log  

- kernel-mshv

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- https://microsoft.visualstudio.com/OS/_workitems/edit/43311210/

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

Built test image and deployed AKS cluster
https://dev.azure.com/mariner-org/mariner/_TestManagement/Runs?runId=80766&_a=runCharts

### CODE REVIEW

**Comment by Redent0r** [comment]
Tested these components and was able to pass all container tests: https://dev.azure.com/mariner-org/mariner/_TestManagement/Runs?runId=80766&_a=runCharts
Additionally performed manual testing and verified all versions are as expected

File: SPECS/kernel-mshv/kernel-mshv.spec [hunk 1]
```diff
@@ -237,6 +237,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_includedir}/perf/perf_dlfilter.h
 
 %changelog
+* Tue Feb 28 2022 Saul Paredes <saulparedes@microsoft.com> - 5.15.92.mshv1-1
```
Comment (sprt) [inline SPECS/kernel-mshv/kernel-mshv.spec:240]:
```suggestion
* Tue Feb 28 2023 Saul Paredes <saulparedes@microsoft.com> - 5.15.92.mshv1-1
```

File: SPECS/kernel-mshv/config [hunk 2]
```diff
@@ -6735,8 +6735,8 @@ CONFIG_LSM="lockdown,yama,loadpin,safesetid,integrity,apparmor,selinux,smack,tom
 # Memory initialization
 #
 CONFIG_INIT_STACK_NONE=y
-CONFIG_INIT_ON_ALLOC_DEFAULT_ON=y
-CONFIG_INIT_ON_FREE_DEFAULT_ON=y
+#CONFIG_INIT_ON_ALLOC_DEFAULT_ON=y is not set
+#CONFIG_INIT_ON_FREE_DEFAULT_ON=y is not set
```
Comment (sprt) [inline SPECS/kernel-mshv/config:6739]:
```suggestion
# CONFIG_INIT_ON_ALLOC_DEFAULT_ON is not set
# CONFIG_INIT_ON_FREE_DEFAULT_ON is not set
```

**Review by sprt — CHANGES_REQUESTED** [review]

File: SPECS/kernel-mshv/config (hunk 2 above)
Comment (Redent0r) [reply SPECS/kernel-mshv/config]: I didn't know you can suggest changes like that. I just had to press commit button for your suggested changes. Really cool!

**Review by sprt — APPROVED** [review]

**Pull request merged**